aea/helpers/async_utils.py

```diff
@@ -412,7 +412,7 @@ def start(self) -> bool:
             self._thread = Thread(
                 target=self._thread_target, name=self.__class__.__name__  # type: ignore # loop was set in set_loop
             )
-            self._thread.setDaemon(True)
+            self._thread.setDaemon(True)  # pylint: disable=deprecated-method
```

Comment:
There was only 1 of these errors, so I skipped it locally.

Reply:
https://github.com/python/cpython/pull/25174

`threading.Thread.setDaemon` => :attr:`threading.Thread.daemon`